worlds/stardew_valley/docs/setup_fr.md

```diff
@@ -0,0 +1,86 @@
+
+Lancez Stardew Valley avec SMAPI. Une fois que vous êtes sur l'écran d'accueil de Stardew Valley, créez une nouvelle ferme.
+
+Sur la page de création d'un nouveau personnage, vous verrez trois nouveaux champs, utilisés pour lier votre nouveau personnage au multiworld d'Archipelago.
+
+![image](https://i.imgur.com/b8KZy2F.png)
+
+Vous pouvez customiser votre ferme et votre personnage autant que vous le souhaitez.
+
+Le champ "Server" doit contenir l'adresse et le port, et le champ "Slot" doit contenir le nom que vous avez spécifié dans votre yaml.
+
+`archipelago.gg:38281`
+
+`StardewPlayer`
+
+Le mot de passe est facultatif.
+
+Votre partie se connectera automatiquement à Archipelago, et se reconnectera automatiquement en chargeant la partie, plus tard.
+
+Vous n'aurez jamais besoin d'entrer ces informations de nouveau pour ce personnage, à moins que vous ne changiez l'adresse ip ou le port de la salle/page.
+Si l'adresse ip ou le port de la salle/page **change**, vous pouvez suivre les instructions suivantes pour modifier les informations de connection de votre partie.
+- Lancez la version mod de Stradew Valley
+- En étant **sur le menu principal** du jeu, entrez la commande suivante **dans la console SMAPI**:
+- `connect_override ip:port slot password`
+- Exemple: `connect_override archipelago.gg:38281 StardewPlayer`
+- Chargez votre partie. Les nouvelles informations de connexion seront utilisées, à la place des anciennes.
+- Jouer un jour complet, dormez, et sauvegardez le jeu. Ces informations de connexions remplaceront les anciennes et deviendront permanentes.
+
+### Intéragir avec le MultiWorld depuis le jeu
+
+Lorsque vous vous connectez, vous devriez voir un message dans le chat vous informant de la commande `!!help`. Cette comande listera d'autres commandes de chat exclusives à Stardew Valley, que vous pouvez utiliser.
+
+En outre, vous pouvez utiliser le chat du jeu pour communiquer à d'autres joueurs du MultiWorld, à condition qu'ils jouent à un jeu ayant la fontion "chat".
+
+Enfin, vous pouvez également utiliser la commande `!help` depuis le chat du jeu, vous permettant de demander des indices sur certains objets, ou des endroits encore non trouvés.
+
+Il est important de noter que le chat de Stardew Valley est assez limité dans ses capacités. Par exemple, il ne permet pas de remonter l'historique, et ainsi de voir ce qui est sorti de l'affichage. La console SMAPI, qui tourne en même temps que votre jeu aura le même historique, entier, et sera sans doute une meilleure option pour lire les ancien messages. 
+Pour une meilleure expérience de chat, vous pouvez aussi utiliser le Client de Texte Archipelago (Archipelago Text Client) officiel; il ne vous permettra cependant pas de taper des commanddes exclusives à Stardew Valley.
+
+### Jouer avec des mods supportés
+
+Voir la [documentation des mods supportés](https://github.com/agilbert1412/StardewArchipelago/blob/4.x.x/Documentation/Supported%20Mods.md)
```

Comment:
Stardew Valley is on 5.x.x now
I don't know if there is an updated list for this version, but if there is, the link should probably be listed here

Reply:
This is not the only place that the document that 4.x.x is mentioned, just the one I saw first